commit 331105fa44a0b03c7f7f5a543b92aab3ec659385
Author: ivan tkachenko <me@ratijas.tk>
Date:   Wed Mar 15 21:43:15 2023 +0600

    FlatpakPermission: Add back no-args default constructor
    
    Older toolchain such as Kubuntu requires default constructor to use this
    class with QVector. Do not use this constructor manually.
    
    BUG: 467399
    (cherry picked from commit 7e644d8f39e38f5713938424fef6064261270023)

diff --git a/flatpakpermission.h b/flatpakpermission.h
index 8f24354..7b9a5c7 100644
--- a/flatpakpermission.h
+++ b/flatpakpermission.h
@@ -99,6 +99,10 @@ public:
         Advanced
     };
 
+    // Default constructor is required for meta-type registration.
+    /** Default constructor. Creates an invalid entry. */
+    FlatpakPermission() = default;
+
     /**
      * Create a Dummy entry for the Advanced and user-editable sections, just so
      * that ListView shows a section header even if there are no permission row
